test/DotnetTektonTasks.Tests/DotnetPublishImageTask.verified.txt

```diff
@@ -0,0 +1,195 @@
+﻿# Source: task-dotnet/templates/dotnet-publish-image.yaml
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: dotnet-publish-image
+  labels:
+    app.kubernetes.io/version: 0.1.0
+  annotations:
+    artifacthub.io/category: integration-delivery
+    artifacthub.io/maintainers: |
+      - name: Red Hat
+        email: dotnet-team@redhat.com
```

Comment:
Is this an email address we should actually advertise?

Reply:
afaik it is public? Is there an other one we can use?